service/src/test/java/bio/terra/cbas/runsets/monitoring/TestSmartRunsPoller.java

```diff
@@ -69,10 +74,13 @@ void pollRunningRuns() throws ApiException {
     when(cromwellService.runStatus(eq(runningRunEngineId)))
         .thenReturn(new RunStatus().runId(runningRunEngineId).state(State.RUNNING));
 
+    when(runsDao.updateLastPolledTimestamp(eq(runToUpdate.id()))).thenReturn(1);
+
     var actual = smartRunsPoller.updateRuns(List.of(runToUpdate, runAlreadyCompleted));
 
     verify(cromwellService).runStatus(eq(runningRunEngineId));
     verify(cromwellService, never()).runStatus(eq(completedRunEngineId));
+    verify(runsDao).updateLastPolledTimestamp(eq(runToUpdate.id()));
```

Comment:
Do we also want to verify that this is _never_ called for `runAlreadyCompleted`?
```suggestion
    verify(runsDao).updateLastPolledTimestamp(eq(runToUpdate.id()));
    verify(runsDao, never()).updateLastPolledTimestamp(eq(runAlreadyCompleted.id()));
```